mobile/lib/resources/fetch_data.dart

```diff
@@ -31,10 +38,38 @@ Future<Iterable<T>> fetchData<T>(
   try {
     final jsonData = jsonDecode(response.body) as List<dynamic>;
     return jsonData.map(
-      (dynamic data) => fromJson(data as Map<String, dynamic>),
+      (data) => fromJson(data as Map<String, dynamic>),
     );
-  } catch (_) {
+  } catch (e) {
```

Comment:
you're not using the `e` anyway, so why change it from `_`?

Reply:
probably forgor